Tests/SwiftPrettyPrintTests/Public/CombineExtensionTests.swift

```diff
@@ -177,6 +226,33 @@ final class CombineExtensionTests: XCTestCase {
             ]
             receive finished
             """)
+        
+        let recorder2 = StringRecorder()
+        
+        subscribeAndWait(
+            Fail<[Int], TestError>(error: TestError()).prettyPrint(to: recorder2).eraseToAnyPublisher()
+        )
+        
+        assertEqualLines(recorder2.contents.split(separator: "\n")[2...].joined(separator: "\n"),
+            """
+            receive failure:
+            TestError(
+                code: 1,
+                message: "This is the error"
+            )
+            """)
```

Comment:
You may also want to use the `do` scope.

```swift
// Success
do {
    let recorder = StringRecorder()
   ...
}

// Failure
do {
    let recorder = StringRecorder()
   ...
}
```

I'll leave it to you.

Reply:
Thanks! 
It's nice.
I used it